go/libkb/syncer.go

```diff
@@ -5,15 +5,16 @@ package libkb
 
 import (
 	"fmt"
+	"sync"
+
 	keybase1 "github.com/keybase/client/go/protocol/keybase1"
 	"golang.org/x/net/context"
-	"sync"
 )
 
 type Syncer interface {
 	Contextifier
 	sync.Locker
-	loadFromStorage(m MetaContext, u keybase1.UID) error
+	loadFromStorage(m MetaContext, u keybase1.UID, useExpiration bool) error
```

Comment:
loadFromStorageUseExpiration(m, u) and loadFromStorageNoExpiration(m, u) might be clearer, but  you don't have to make that change, just a thought.